command 'python C:\Users\z3531278\Documents\01_FEWS-RegionHome-Aus\Modules\NSSDownload\python\retrieveNSS.py C:\Users\z3531278\Documents\01_FEWS-RegionHome-Aus\Modules\NSSDownload 20210910_0000 C:\Users\z3531278\Documents\01_FEWS-RegionHome-Aus\' return with error (code 1): b'Traceback (most recent call last):\r\n  File "C:\\Users\\z3531278\\Documents\\01_FEWS-RegionHome-Aus\\Modules\\NSSDownload\\python\\retrieveNSS.py", line 118, in <module>\r\n    main()\r\n  File "C:\\Users\\z3531278\\Documents\\01_FEWS-RegionHome-Aus\\Modules\\NSSDownload\\python\\retrieveNSS.py", line 50, in main\r\n    fcst = pickle.load(open(os.path.join(forecastDir,"forecast.pkl"),"rb"))\r\nFileNotFoundError: [Errno 2] No such file or directory: \'C:\\\\Users\\\\z3531278\\\\Documents\\\\01_FEWS-RegionHome-Aus\\\\Modules\\\\NSSDownload\\\\Forecasts\\\\20210910_0000\\\\forecast.pkl\'\r\n\r\nTraceback (most recent call last):\r\n  File "C:\\Users\\z3531278\\Documents\\01_FEWS-RegionHome-Aus\\Modules\\NSSDownload\\python\\retrieveNSS.py", line 118, in <module>\r\n    main()\r\n  File "C:\\Users\\z3531278\\Documents\\01_FEWS-RegionHome-Aus\\Modules\\NSSDownload\\python\\retrieveNSS.py", line 50, in main\r\n    fcst = pickle.load(open(os.path.join(forecastDir,"forecast.pkl"),"rb"))\r\nFileNotFoundError: [Errno 2] No such file or directory: \'C:\\\\Users\\\\z3531278\\\\Documents\\\\01_FEWS-RegionHome-Aus\\\\Modules\\\\NSSDownload\\\\Forecasts\\\\20210910_0000\\\\forecast.pkl\'\r\n'Traceback (most recent call last):
  File "C:\Users\z3531278\Documents\01_FEWS-RegionHome-Aus\Scripts\runForecast\run_forecast_loop.py", line 70, in runModule
    subprocess.check_output(command,shell=True,stderr=subprocess.STDOUT)
  File "C:\Users\z3531278\Documents\01_FEWS-RegionHome-Aus\bin\windows\python\bin\conda-venv\lib\subprocess.py", line 415, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
  File "C:\Users\z3531278\Documents\01_FEWS-RegionHome-Aus\bin\windows\python\bin\conda-venv\lib\subprocess.py", line 516, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command 'python C:\Users\z3531278\Documents\01_FEWS-RegionHome-Aus\Modules\NSSDownload\python\retrieveNSS.py C:\Users\z3531278\Documents\01_FEWS-RegionHome-Aus\Modules\NSSDownload 20210910_0000 C:\Users\z3531278\Documents\01_FEWS-RegionHome-Aus\' returned non-zero exit status 1.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\z3531278\Documents\01_FEWS-RegionHome-Aus\Scripts\runForecast\run_forecast_loop.py", line 238, in <module>
    main()
  File "C:\Users\z3531278\Documents\01_FEWS-RegionHome-Aus\Scripts\runForecast\run_forecast_loop.py", line 158, in main
    runModule(script=retrieveNSSPy,args=arguments)
  File "C:\Users\z3531278\Documents\01_FEWS-RegionHome-Aus\Scripts\runForecast\run_forecast_loop.py", line 76, in runModule
    raise RuntimeError("command '{}' return with error (code {}): {}".format(e.cmd, e.returncode, e.output))
RuntimeError: command 'python C:\Users\z3531278\Documents\01_FEWS-RegionHome-Aus\Modules\NSSDownload\python\retrieveNSS.py C:\Users\z3531278\Documents\01_FEWS-RegionHome-Aus\Modules\NSSDownload 20210910_0000 C:\Users\z3531278\Documents\01_FEWS-RegionHome-Aus\' return with error (code 1): b'Traceback (most recent call last):\r\n  File "C:\\Users\\z3531278\\Documents\\01_FEWS-RegionHome-Aus\\Modules\\NSSDownload\\python\\retrieveNSS.py", line 118, in <module>\r\n    main()\r\n  File "C:\\Users\\z3531278\\Documents\\01_FEWS-RegionHome-Aus\\Modules\\NSSDownload\\python\\retrieveNSS.py", line 50, in main\r\n    fcst = pickle.load(open(os.path.join(forecastDir,"forecast.pkl"),"rb"))\r\nFileNotFoundError: [Errno 2] No such file or directory: \'C:\\\\Users\\\\z3531278\\\\Documents\\\\01_FEWS-RegionHome-Aus\\\\Modules\\\\NSSDownload\\\\Forecasts\\\\20210910_0000\\\\forecast.pkl\'\r\n\r\nTraceback (most recent call last):\r\n  File "C:\\Users\\z3531278\\Documents\\01_FEWS-RegionHome-Aus\\Modules\\NSSDownload\\python\\retrieveNSS.py", line 118, in <module>\r\n    main()\r\n  File "C:\\Users\\z3531278\\Documents\\01_FEWS-RegionHome-Aus\\Modules\\NSSDownload\\python\\retrieveNSS.py", line 50, in main\r\n    fcst = pickle.load(open(os.path.join(forecastDir,"forecast.pkl"),"rb"))\r\nFileNotFoundError: [Errno 2] No such file or directory: \'C:\\\\Users\\\\z3531278\\\\Documents\\\\01_FEWS-RegionHome-Aus\\\\Modules\\\\NSSDownload\\\\Forecasts\\\\20210910_0000\\\\forecast.pkl\'\r\n'
